<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDataWidget</class>
 <widget class="QWidget" name="NewDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="new_data_type_combo">
       <item>
        <property name="text">
         <string>Event</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Interval</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analog Time Series</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Point</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mask</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tensor</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="new_data_name"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="new_data_button">
     <property name="text">
      <string>Create New Data</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
